Graphics/GraphicsEngine/src/TextureBase.cpp

```diff
@@ -100,7 +101,7 @@ void ValidateTextureDesc(const TextureDesc& Desc, const IRenderDevice* pDevice)
             LOG_TEXTURE_ERROR_AND_THROW("Texture cube/cube array must have at least 6 slices (", Desc.ArraySize, " provided).");
     }
 
-    Uint32 MaxDim = 0;
+    MAYBE_UNUSED Uint32 MaxDim = 0;
```

Comment:
Why?
Any other solutions to avoid MAYBE_UNUSED ?

Reply:
We read MaxDim only inside VERIFY, and VERIFY in the Release build is empty.
It seems we should explicitly check as in the code below